README.md

```diff
@@ -1,24 +1,103 @@
-# README
+<div align="center">
+  <h1><b>Blog App</b></h1>
+</div>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Ruby Capstone - Catalog of My Things <a name="about-project"></a>
 
-* System dependencies
+Is a blog app that is built with ruby on rails
 
-* Configuration
+### Tech Stack <a name="tech-stack"></a>
 
-* Database creation
+Ruby on Rails
 
-* Database initialization
+### Key Features <a name="key-features"></a>
 
-* How to run the test suite
+- **postgres database, tables created**
 
-* Services (job queues, cache servers, search engines, etc.)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Deployment instructions
+## 🚀 Video Demo <a name="live-demo"></a>
 
-* ...
+- not yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- not yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone project
+
+```bash
+git clone https://github.com/david63011/Blog-App-Rails.git
+
+cd MyApp
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **David Underwood**
+
+- GitHub: [@david63011](https://github.com/david63011)
+- Twitter: [@david630](https://twitter.com/david630)
+- LinkedIn: [David Underwood](https://www.linkedin.com/in/davidunder)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **More methods**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star 😁🌟✨
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Great job working on this milestone. Kindly add the steps to follow to run your project locally as well as to run your spec tests.

Reply:
ok fixed